include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -61,20 +60,20 @@ namespace detail { namespace overlay
 
 struct policy_verify_nothing
 {
-    static bool const use_side_verification = false;
-    static bool const use_start_turn = false;
-    static bool const use_handle_as_touch = false;
-    static bool const use_handle_as_equal = false;
-    static bool const use_handle_imperfect_touch = false;
+    static bool constexpr use_side_verification = false;
+    static bool constexpr use_start_turn = false;
+    static bool constexpr use_handle_as_touch = false;
+    static bool constexpr use_handle_as_equal = false;
+    static bool constexpr use_handle_imperfect_touch = false;
 };
 
 struct policy_verify_all
 {
-    static bool const use_side_verification = true;
-    static bool const use_start_turn = true;
-    static bool const use_handle_as_touch = true;
-    static bool const use_handle_as_equal = true;
-    static bool const use_handle_imperfect_touch = true;
+    static bool constexpr use_side_verification = true;
+    static bool constexpr use_start_turn = true;
+    static bool constexpr use_handle_as_touch = true;
+    static bool constexpr use_handle_as_equal = true;
+    static bool constexpr use_handle_imperfect_touch = true;
```

Comment:
Is `static constexpr` valid? I didn't check but I remembered there was something with it.
Why is it actually changed? (A big PR can better have exactly one subject)

Reply:
But anyway, if it is valid, it is fine.
Thanks for the PR, it seems a huge effort and a huge style improvement.

Reply:
I think, it changes the meaning in so far as that these values will not have actual storage, which I think is appropriate in this context, since I understood that these are never used in ways that requires taking their addresses but rather for decisions that can all be resolved at compile time. But I can just take it out of this PR tomorrow to have a more clear scope.